app.js

```diff
@@ -51,6 +51,7 @@ app.use(cookieParser());
 // Nunjucks configuration for application
 const appViews = [
   path.join(__dirname, 'app/views/'),
+  path.join(__dirname, 'node_modules/nhsuk-frontend/packages/macros'),
```

Comment:
I think this should probably be sorted alphabetically.

Reply:
Done in 2e2d2150ece1b7fc9d17508680f93c3b81455f0f